### PR TITLE
feat: lod generator

### PR DESCRIPTION
**Pull request by dalkia**

### CODE REVIEW

File: asset-bundle-converter/Assets/AssetBundleConverter/Editor/ExportFBXAssetBundles.cs [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using DCL.ABConverter;
+using DCL.Shaders;
+using UnityEngine.Experimental.Rendering;
+using UnityEngine.Rendering;
+using Object = UnityEngine.Object;
+using Random = UnityEngine.Random;
+
+public class ExportFBXAssetBundles : MonoBehaviour
```
Comment (aleortega) [inline asset-bundle-converter/Assets/AssetBundleConverter/Editor/ExportFBXAssetBundles.cs:14]: Can we add this to a namespace similar to this one? 
https://github.dev/decentraland/asset-bundle-converter/blob/main/asset-bundle-converter/Assets/AssetBundleConverter/SceneClient.cs

Comment (dalkia) [reply asset-bundle-converter/Assets/AssetBundleConverter/Editor/ExportFBXAssetBundles.cs:14]: Done!

File: asset-bundle-converter/Assets/AssetBundleConverter/LODsConverter/Editor/LODClient.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    public class LODClient : MonoBehaviour
+    {
+
+        [MenuItem("Assets/Export URL LODs")]
+        public static void ExportURLLODsToAssetBundles()
+        {
+            string[] commandLineArgs = Environment.GetCommandLineArgs();
+            
+            string customOutputDirectory = "";
+            string lodsURL = "https://lods-bucket-ed4300a.s3.amazonaws.com/-17,-21/LOD/Sources/1707776785658/bafkreidnwpjkv3yoxsz6iiqh3fahuec7lfsqtmkyz3yf6dgps454ngldnu_0.fbx;" +
```
Comment (Kinerius) [inline asset-bundle-converter/Assets/AssetBundleConverter/LODsConverter/Editor/LODClient.cs:22]: do we want these URLs to be public?

**Review by Kinerius — APPROVED** [review]
LGTM

File: asset-bundle-converter/Assets/AssetBundleConverter/LODsConverter/Editor/LODClient.cs (hunk 2 above)
Comment (dalkia) [reply asset-bundle-converter/Assets/AssetBundleConverter/LODsConverter/Editor/LODClient.cs:22]: Ideally, no. We tried integrating the amazon api to make them private; but due to time constraints, we dropped it. We are looking forward to make the downloader using the amazon api in the hardening week. 

Then, they will be private

**Pull request merged**